upgraderesponder/service.go

```diff
@@ -65,50 +71,26 @@ type Location struct {
 	} `json:"country"`
 }
 
-type ResponseConfig struct {
-	Versions []Version
-}
-
-type Version struct {
-	Name                 string // must be in semantic versioning
-	ReleaseDate          string
-	MinUpgradableVersion string // can be empty or semantic versioning
-	Tags                 []string
-	ExtraInfo            map[string]string
-}
-
-type CheckUpgradeRequest struct {
-	AppVersion string            `json:"appVersion"`
-	ExtraInfo  map[string]string `json:"extraInfo"`
-}
-
 type CheckUpgradeResponse struct {
-	Versions                 []Version `json:"versions"`
-	RequestIntervalInMinutes int       `json:"requestIntervalInMinutes"`
+	Versions                 []rd.Version `json:"versions"`
+	RequestIntervalInMinutes int          `json:"requestIntervalInMinutes"`
```

Comment:
Is that a hidden tab space?

Reply:
Maybe? What makes you think that? This would have been auto-formatted by my editor.